src/transformers/cache_utils.py

```diff
@@ -431,12 +431,12 @@ def update(
             if len(self.key_cache) <= layer_idx:
                 # There may be skipped layers, fill them with empty lists
                 for _ in range(len(self.key_cache), layer_idx):
-                    self.key_cache.append([])
-                    self.value_cache.append([])
+                    self.key_cache.append(torch.tensor([]))
```

Comment:
an empty tensor is indeed more in line with the whole docs and expected interface, thank you for updating 👍